### ISSUE TITLE
Documentation in Kindle format

### ISSUE THREAD

**Issue by AlexVPopov** [pakyow/docs#16]
I think it would be really cool if this documentation is also available in .mobi format, similarly to the Rails guides.

**Comment by bryanp** [comment]
Great idea! Do you know how to convert .html to .mobi? Could probably build this in fairly easily.

**Comment by AlexVPopov** [comment]
No, but the guys who work on the Rails guides do. I'll ask them.
